package.json

```diff
@@ -10,6 +10,7 @@
   ],
   "resolutions": {
     "@optimizely/js-sdk-utils": "0.4.0",
+    "@types/react": "18.2.6",
```

Comment:
Updates the `types/react` of any dependency to ensure all dependencies are using the same type checking for React components.